randovania/games/prime1/exporter/patch_data_factory.py

```diff
@@ -555,11 +555,13 @@ def pick_random_dst(src_name, src_dock):
 
         boss_sizes = None
         if self.configuration.random_boss_sizes:
-            def get_random_size(min, max):
+            def get_random_size(minimum, maximum):
                 if self.rng.choice([True, False]):
-                    return self.rng.uniform(min, 0.8)
+                    temp = [self.rng.uniform(minimum, 1.0), self.rng.uniform(minimum, 1.0)]
```

Comment:
taking 2RN so it bias towards smallers numbers?

Reply:
Correct. In testing "real random" and "`< 0.8` / `> 1.2` random" felt less exciting.